packages/design-system/src/atoms/tooltip/index.web.tsx

```diff
@@ -61,33 +64,39 @@ export const Tooltip: React.FC<TooltipProps> = ({
     useFocus(context),
   ])
 
+  const wrapper = useCallback((render: React.ReactNode) => {
+    return <FloatingPortal>{render}</FloatingPortal>
+  }, [])
+
```

Comment:
You can declare it outside of the component context, so we save memory (no `useCallback` => no memoization)